ui/src/pages/version/components/forms/components/docker_config/DockerDeploymentPanel.js

```diff
@@ -0,0 +1,99 @@
+import React, { useContext } from "react";
+import {
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiFormRow,
+  EuiSpacer
+} from "@elastic/eui";
+import {
+  FormLabelWithToolTip,
+  SelectDockerImageComboBox,
+  useOnChangeHandler
+} from "@gojek/mlp-ui";
+import { appConfig } from "../../../../../../config";
+import DockerRegistriesContext from "../../../../../../providers/docker/context";
+import { Panel } from "../Panel";
+
+import "./DockerDeploymentPanel.scss";
+
+const imageOptions = [];
+
+export const DockerDeploymentPanel = ({
+  values: { image, command, args },
+  onChangeHandler,
+  errors = {}
+}) => {
+  const { onChange } = useOnChangeHandler(onChangeHandler);
+  const registries = useContext(DockerRegistriesContext);
+
+  return (
+    <Panel title="Deployment">
+      <EuiForm>
+        <EuiFormRow
+          label="Docker Image *"
+          isInvalid={!!errors.image}
+          error={errors.image}
+          fullWidth
+          display="row">
+          <SelectDockerImageComboBox
+            fullWidth
+            value={image || `${appConfig.defaultDockerRegistry}/`}
+            placeholder="Docker image name and tag"
+            registryOptions={registries}
+            imageOptions={imageOptions}
+            onChange={onChange("image")}
+            isInvalid={!!errors.image}
+          />
+        </EuiFormRow>
+
+        <EuiSpacer size="m" />
+
+        <EuiFlexGroup direction="column" gutterSize="m">
+          <EuiFlexItem grow={1}>
```

Comment:
what does 1 represent for? true?

Reply:
It represents the width of the flex item -- https://elastic.github.io/eui/#/layout/flex#proportional-widths-of-items

But I just realized we don't need to set it that way because we already set EuiFlexGroup's direction to "column". I'll update it.